migrations/20200401160718_migrate_fate_of_animals_killed.js

```diff
@@ -0,0 +1,60 @@
+const { uniq } = require('lodash');
+
+const transform = version => {
+  if (!version) {
+    return;
+  }
+  if (version['fate-of-animals-nts'] === false) {
+    version['fate-of-animals'] = uniq([
+      ...(version['fate-of-animals'] || []),
+      'killed'
+    ]);
+  }
+  return version;
+};
+
+exports.transform = transform;
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => {
+      return knex
+        .select('id')
+        .from('projects')
+        .where({ 'schema_version': 1 })
+    })
+    .then(projects => {
+      return projects.reduce((promise, project) => {
+        return promise
+          .then(() => {
+            return knex
+              .select('id', 'data')
```

Comment:
Since this doesn't actually use the top-level project, and only queries the projects to get the `schemaVersion: 1` ones, can you make it do a join in the same way as #290 so it's not having to hold multiple project versions in memory at once?

Also, can you add some log output around the total number found at the start, and progress as each is updated?

Thanks.